File: tjhsst comp sci 2/pyKarel Student Code/Lab03 Redone/Lab03.py
#Name:Karthik and Vivian
#Date: 9/4/2019

from pyKarel import *
from Climber import Climber
wld = World('mountain',height =8,width=16)


t = Climber(wld, 8,1, north, 1)
r= Climber(wld, 8,1, north, 1)

r.putBeeper()
r.turnRight()
t.turnRight()
r.move()
t.move()
r.climbUpRight()
t.climbUpRight()
r.climbUpRight()
t.climbUpRight()
r.climbUpRight()
t.climbUpRight()
r.turnRight()
t.turnRight()
r.climbDownRight()
t.climbDownRight()
r.climbDownRight()
t.climbDownRight()
r.pickBeeper()
t.turnAround()
r.turnAround()
t.move()
t.move()
r.move()
r.move()
t.climbUpLeft()
r.climbUpLeft()
t.turnLeft()
t.move()
t.move()
r.turnLeft()
r.move()
r.move()
t.climbDownLeft()
r.climbDownLeft()
t.climbDownLeft()
r.climbDownLeft()
t.climbDownLeft()
r.climbDownLeft()
t.putBeeper()
r.move()
t.move()




wld.mainloop()

File: tjhsst comp sci 2/Java/Units/Unit1/Lab03.java
//Name: Vivian Feng and Shriya Muthukumar Date: March 16,2020

import edu.fcps.karel2.Display;
import edu.fcps.karel2.Robot;

public class Lab03
{
   public static void main(String[] args){
      Display.openWorld("maps/mountain.map");
      Display.setSize(16,16);
      Climber t = new Climber(8);
      Climber r= new Climber(8);
      
      r.putBeeper();
      r.turnRight();
      t.turnRight();
      r.move();
      t.move();
      r.climbUpRight();
      t.climbUpRight();
      r.climbUpRight();
      t.climbUpRight();
      r.climbUpRight();
      t.climbUpRight();
      
      t.climbDownRight();
      r.climbDownRight();
      t.climbDownRight();
      r.climbDownRight();
      
      r.pickBeeper();
      r.turnAround();
      t.turnAround();
      t.climbUpLeft();
      r.climbUpLeft();
      t.climbUpLeft();
      r.climbUpLeft();
      t.climbDownLeft();
      r.climbDownLeft();
      t.climbDownLeft();
      r.climbDownLeft();
      t.climbDownLeft();
      r.climbDownLeft();
      t.move();
      r.move();
      t.putBeeper();
      r.move();
      t.move();
   }
}